Demo/app/src/main/java/com/krunal/demo/kotlinPractice/visibilitymod/Unrelated.kt

```diff
@@ -0,0 +1,12 @@
+package com.krunal.demo.kotlinPractice.visibilitymod
+
+class Unrelated(outer: Outer) {
+
+    init {
+//        outer.a Cannot access 'a': it is private in 'Outer'
```

Comment:
remove the commented code

Reply:
commented lines in this file is to understand that which members are accessible and which are not.